testnets/wavehashtestnet/chain.json

```diff
@@ -0,0 +1,113 @@
+{
+    "$schema": "../../chain.schema.json",
+    "chain_name": "wavehashtestnet",
+    "status": "live",
+    "network_type": "testnet",
+    "pretty_name": "wavehash Testnet",
+    "chain_id": "testnet-1",
+    "bech32_prefix": "wavehash",
+    "daemon_name": "wavehashd",
+    "node_home": "$HOME/.wavehash",
+    "key_algos": [
+      "secp256k1"
+    ],
+    "slip44": 118,
+    "fees": {
+      "fee_tokens": [
+        {
+          "denom": "uwahax",
+          "low_gas_price": 0.0025,
+          "average_gas_price": 0.0025,
+          "high_gas_price": 0.0024
+        }
+      ]
+    },
+    "staking": {
+      "staking_tokens": [
+        {
+          "denom": "uwahax"
+        }
+      ]
+    },
+    "codebase": {
+      "git_repo": "https://github.com/WaveHashProtocol/wavehash",
+      "recommended_version": "v6.0.0",
+      "compatible_versions": [
+        "v6.0.0"
+      ],
+      "cosmos_sdk_version": "0.45",
+      "consensus": {
+        "type": "tendermint",
+        "version": "0.34"
+      },
+      "cosmwasm_version": "0.30",
+      "cosmwasm_enabled": true,
+      "ibc_go_version": "4.3.0",
+      "genesis": {
+        "genesis_url": "https://raw.githubusercontent.com/WaveHashProtocol/testnet/main/testnet-1/genesis.json"
+      },
+      "versions": [
+        {
+          "name": "v6.0.0",
```

Comment:
you seem to have v6.0.0 in the versions array twice. Please only have one object for v6.0.0 (but you may define previous versions here)